.ci/no-exception-test.sh

```diff
@@ -53,6 +53,28 @@ guava-with-sun-checks)
   rm sun_checks.*
   ;;
 
+openjdk14-with-checks-nonjavadoc-error)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: $CS_POM_VERSION
+  mvn -e clean install -Pno-validations
```

Comment:
Checkstyle is built already: https://github.com/checkstyle/checkstyle/blob/master/.semaphore/semaphore.yml#L88

Reply:
i'll remove duplicate from swmaphore.yml

Reply:
this is a prat I still do not have strong position, where to keep build of checkstyle in validation commands or in CI config (presumably it is "given").
Some CI reuse the same host for all executions (wercker for example).
probably we can have shells like:
`ci.yml -> your-ci.sh TASK_NAME -> "mvn install ...."  && validation.sh TASKNAME `

Reply:
From above:
>one day this execution will fail in CI in PR, contributor might not be very experienced in CI to figureout how to reproduce it on local, so we will share with him link to single line in config for simple copy and paste in terminal, and it works.

If we offload the building of checkstyle to CI/host and remove it from script, it will not be a 'single line in config for simple copy and paste in terminal' anymore.

Reply:
in my case it is simple copy-paste with presumption that sources are built.
if no built done, failure is obvious
when  no re-build is done, significant execution time reduction.